modules/proc-switching-bf2-nic.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * networking/switching-bf2-nic-dpu.adoc
+
+:_content-type: PROCEDURE
+[id="proc-switching-bf2-nic_{context}"]
+= Switching Bluefield-2 from DPU mode to NIC mode
+
+Use the following procedure to switch Bluefield-2 from data processing units (DPU) mode to network interface controller (NIC) mode.  
+
+[IMPORTANT]
+====
+Currently, only switching Bluefield-2 from DPU to NIC mode is supported. Switching from NIC mode to DPU mode is unsupported. 
+====
+
+.Prerequisites
+
+* You have installed the SR-IOV Operator. For more information, see "Installing SR-IOV Network Operator". 
+* You have updated Bluefield-2 to the latest firmware. For more information, see link:https://network.nvidia.com/support/firmware/bluefield2/[Firmware for NVIDIA BlueField-2]. 
+
+.Procedure
+
+. Add the following labels to each of your worker nodes by entering the following commands:
++
+[source,terminal]
+----
+$ oc label node <example_node_name_one> node-role.kubernetes.io/sriov=
+----
++
+[source,terminal]
+----
+$ oc label node <example_node_name_two> node-role.kubernetes.io/sriov=
+
+----
+
+. Create a machine config pool for the SR-IOV Operator, for example: 
++
+[source,terminal]
+----
+
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: sriov
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,sriov]}
+  nodeSelector:
+    matchLabels:
+            node-role.kubernetes.io/sriov: ""
+----
+
+. Apply the following `machineconfig.yaml` file to the worker nodes:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: sriov
+  name: 99-bf2-dpu
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,ZmluZF9jb250YWluZXIoKSB7CiAgY3JpY3RsIHBzIC1vIGpzb24gfCBqcSAtciAnLmNvbnRhaW5lcnNbXSB8IHNlbGVjdCgubWV0YWRhdGEubmFtZT09InNyaW92LW5ldHdvcmstY29uZmlnLWRhZW1vbiIpIHwgLmlkJwp9CnVudGlsIG91dHB1dD0kKGZpbmRfY29udGFpbmVyKTsgW1sgLW4gIiRvdXRwdXQiIF1dOyBkbwogIGVjaG8gIndhaXRpbmcgZm9yIGNvbnRhaW5lciB0byBjb21lIHVwIgogIHNsZWVwIDE7CmRvbmUKISBzdWRvIGNyaWN0bCBleGVjICRvdXRwdXQgL2JpbmRhdGEvc2NyaXB0cy9iZjItc3dpdGNoLW1vZGUuc2ggIiRAIgo=
```

Comment:
@bn222 Should we include in the docs what this base64 encoded script is? Just for reference to users such that they don't ask questions?

Reply:
No, I don't think we should spend more time on this.

Reply:
Agreed